core/src/main/java/apoc/math/Maths.java

```diff
@@ -69,5 +69,60 @@ public Long minByte(){
         return Long.valueOf(Byte.MIN_VALUE);
     }
 
+    @UserFunction
```

Comment:
not sure if we should NaN (e.g division by zero) just bubble up or convert them to null?

Reply:
@jexp but in this case the solution for `1.0 + Math.exp(-x)=0` is non-real, how should we manage it?